src/Sylius/Bundle/ResourceBundle/Controller/RequestConfigurationFactory.php

```diff
@@ -74,12 +74,18 @@ private function parseApiParameters(Request $request)
     {
         $parameters = [];
 
-        if (preg_match(self::API_VERSION_REGEXP, $request->headers->get(self::API_VERSION_HEADER), $matches)) {
-            $parameters['serialization_version'] = $matches['version'];
+        $apiVersionHeaders = $request->headers->get(self::API_VERSION_HEADER, null, false);
+        foreach ($apiVersionHeaders as $apiVersionHeader) {
+            if (preg_match(self::API_VERSION_REGEXP, $apiVersionHeader, $matches)) {
+                $parameters['serialization_version'] = $matches['version'];
```

Comment:
I've approved it too quickly. According to [this issue](https://github.com/symfony/symfony/issues/16646) and [this question on SO](https://stackoverflow.com/questions/39912920/how-to-interpret-multiple-accept-headers) we should rather combine returned values instead of honoring only the last one.